src/components/Tabs/index.examples.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import Tabs from '.'
+import Card from '../Card'
+
+storiesOf('Tabs')
+  .addWithInfo(
+    'Info',
+    'Used to display sections of components',
+    () => (
+      <Tabs groups={[
+        {
+          title: 'Tab 1',
+          component: (
+            <Card>
+              <h1>Tab Content 1</h1>
+            </Card>
+          )
+        },
+        {
+          title: 'Tab 2',
+          component: (
+            <Card>
+              <h1>Tab Content 2</h1>
+            </Card>
+          )
+        }
+      ]}
+      />
+    )
+  )
```

Comment:
Would rather not have the `Card` wrapper either as it makes the examples look like that is a part of the component. The `NodeFixture` is meant to be a generic "put anything here" example.